### ISSUE TITLE
Order of post description information causes poor user experience--subreddit should be first.

### ISSUE THREAD

**Issue by Xenataur** [laurencedawson/reddit-sync-development#1899]
Similar to: https://github.com/laurencedawson/reddit-sync-development/issues/1280

I'm a long time reddit user and a user experience designer. 

The information organization should support quickly scanning the most important information when scrolling through /r/all or their front page. But what is the most important? Let's take each bit of content one at a time in their current order:

- Post title
 - This answers "What am I looking at?" and is of course, the most important information about a post.
- Post tag
 - Many subreddits use tags as categories, giving users contextual meta information about the post they're looking at, so they can better process the post title and decide if it's something they're interested in clicking on. Other subreddits use it to correct changed link titles (/r/politics) or warn users that /r/all has come a knockin'.
- Author (removable via settings)
 - This answers "who posted this?", an unimportant and rarely asked question. Reddit is a link aggregator where the content is the star, not the poster. Aside from some power users looking to avoid /u/GallowBoob content, there is no need for this info on the front page, and Sync smartly lets users exclude it.
- Timestamp
 - This answers "when was it posted?" This is useful but not essential meta information, as it allows users to know how "fresh" the content is, and identify hot topics with high traffic over a short period of time.
- Subreddit (emphasized by styling)
 - This answers "Where was this posted?", second in importance to only the post title (I could be convinced it's more important). A post's subreddit is *essential* contextual meta information as it answers additional questions about the audience for any given post, "What type of user was this intended for?", "What type of user will be posting in the comments?" and additional questions about community culture, "Is this a meta reference?", "Is this a joke title?"
- URL
 - This answers "Where is this content from?", which is important for deciding whether they like or can trust a source.

Taking this into consideration, we can say that the content hierarchy should be:

1. Post title
3. Subreddit
2. Tag
3. Link
4. Timestamp
5. Author 

And given the [copious research that indicates users read in an F shape](https://www.nngroup.com/articles/f-shaped-pattern-reading-web-content) and the scanning, scrolling, low engagement activity that is scrolling through the Reddit front page, it should be organized as such:

Post title [Tag] (Link)
Subreddit • Timestamp • Author

The new Reddit desktop design actually gets this right:
![image](https://user-images.githubusercontent.com/7465285/40068954-bb9edc68-581e-11e8-81b6-a3642b78222e.png)

**tl;dr** Users read the front page by scanning the left most content. Because the subreddit is extremely important context for a post, it should be presented first, as the left-most item in the description.
